packages/easy-coding-standard/config/config.yaml

```diff
@@ -1,24 +0,0 @@
-imports:
-    - { resource: '../packages/**/config/services.yaml' }
-    - { resource: 'services.yaml' }
-
-parameters:
-    # defaults for "PhpCsFixer\WhitespacesFixerConfig"
-    indentation: 'spaces'
-    line_ending: !php/const PHP_EOL
```

Comment:
@weaverryan This one is a somethign I missed :fearful: 

It took me 40 minutes to hack around it